doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -166,31 +179,31 @@ Kubernetes objects and their desired state.
    * - Pod and workloads
      - Description
    * - `Pod <https://kubernetes.io/docs/concepts/workloads/pods/>`__
-     - Pod is a collection of containers that can run on a node. This resource is created by clients and scheduled onto
-       nodes.
+     - A pod is a collection of containers that can run on a node. This resource is created by clients
+       and scheduled onto nodes.
    * - `ReplicaSet <https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/>`__
-     - ReplicaSet ensures that a specified number of pod replicas are running at any given time.
+     - A ReplicaSet ensures that a specified number of pod replicas are running at any given time.
    * - `Deployment <https://kubernetes.io/docs/concepts/workloads/controllers/deployment/>`__
-     - Deployment enables declarative updates for Pods and ReplicaSets.
+     - A deployment enables declarative updates for pods and ReplicaSets.
    * - `DaemonSet <https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/>`__
-     - A Daemon set ensures that the correct nodes run a copy of a Pod.
+     - A DaemonSet ensures that the correct nodes run a copy of a pod.
    * - `Job <https://kubernetes.io/docs/concepts/workloads/controllers/job/>`__
-     - A Job represent a task, it creates one or more Pods and will continue to retry until the expected number of
-       successful completions is reached.
+     - A job represents a task. It creates one or more pods and will continue to retry until the
```

Comment:
```suggestion
     - A job represents a task that creates one or more pods and ensures that the
```

Reply:
I would be inclined to keep the separate sentences " A job represents a task. It creates...". But I agree with the second correction.